javascript/javascript2/week3/jokecreator.js

```diff
@@ -0,0 +1,16 @@
+const logFunnyJoke = () => {
+  console.log("funny joke");
+};
+const logBadJoke = () => {
+  console.log("bad joke");
+};
+
+function jokeCreator(shouldTellFunnyJoke, logFunnyJoke, logBadJoke) {
+  if ((shouldTellFunnyJoke = true)) {
```

Comment:
this is an assignment statement. If you want to check if the variable is true you can just do 
```suggestion
  if (shouldTellFunnyJoke) {
```